ItransitionCourseProject/ViewModels/LoginViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace ItransitionCourseProject.ViewModels
+{
+    public class LoginViewModel
+    {
+        [Required]
+        [Display(Name = "UserName")]
+        public string UserName { get; set; }
```

Comment:
What happens if you delete [Display (Name = "UserName")]? The property already has the same name